client/src/components/new-request/analyze-results.vue

```diff
@@ -490,23 +489,31 @@ export default class AnalyzeResults extends Vue {
       return {
         class: 'approved',
         icon: 'mdi-check-circle',
-        text: 'Name Available',
+        text: 'Name Ready for Review',
         showNextLines: true
       }
     }
     let { length } = this.json.issues
+    if (length >= 1) {
```

Comment:
Is this true for all issue types. Are there any issue types for which we display just a message to the user and still show the proceed option

Reply:
I'm really not sure 😬  
I'll review the designs i have, but every issue seems to require some sort of action, in my limited* experience.

Reply:
There is some more flow changes with regards to names that are not analyzed, which we discovered as a result of this. But there is no action required here, as it is it's own can of worms to address in a new ticket 😁